### ISSUE TITLE
Image upload not working

### ISSUE THREAD

**Issue by snowformatics** [imagekit-developer/imagekit-python#37]
Hi,

I am trying to upload my image:

```
import base64
import os
import sys
from imagekitio import ImageKit
from imagekitio.models.UploadFileRequestOptions import UploadFileRequestOptions

PRIVATE_KEY = os.environ.get('PRIVATE_KEY')
PUBLIC_KEY = os.environ.get('PUBLIC_KEY')

imagekit = ImageKit(
    private_key='private',
    public_key='public',
    url_endpoint='https://ik.imagekit.io/nb4gbrqqe'
)

url = "event20230225_133517_27.jpg"
upload = imagekit.upload(
    file=open(url, "rb"),
    file_name="test.jpg",
    options=UploadFileRequestOptions(
        tags = ["test"]
    )
)

list_files = imagekit.list_files()

print (list_files.response_metadata.raw)
```

But imagekitio.io ist showing just:

![image](https://user-images.githubusercontent.com/5192670/221376632-e3c1d691-da16-4e75-a510-d0f8a0554c94.png)

When I extract the metadata, it looks like the image was not uploaded as image type:

`{'type': 'file', 'name': 'test_doAbVVtZJ.jpg', 'createdAt': '2023-02-25T19:42:54.915Z', 'updatedAt': '2023-02-25T19:42:55.118Z', 'fileId': '63fa64bee809dd54b091d551', 'tags': ['test'], 'AITags': None, 'versionInfo': {'id': '63fa64bee809dd54b091d551', 'name': 'Version 1'}, 'embeddedMetadata': {}, 'customCoordinates': None, 'customMetadata': {}, 'isPrivateFile': False, 'url': 'https://ik.imagekit.io/nb4gbrqqe/test_doAbVVtZJ.jpg', 'thumbnail': 'https://ik.imagekit.io/demo/img/static-file-1.png', 'fileType': 'non-image', 'filePath': '/test_doAbVVtZJ.jpg', 'size': 90}]`

My manually uploaded image show 'fileType': 'image'.

How can I upload my images as image type?

Thanks in adavance
Stefanie

**Comment by Hexvortex** [comment]
Yeah i am also getting broken image after uploading via python script.

**Comment by samims** [comment]
As mentioned in the test, the file needs to be base64 encoded. 
https://github.com/imagekit-developer/imagekit-python/blob/master/tests/test_files_ops.py#L154

**Comment by snowformatics** [comment]
Works, thanks!

**Comment by reisenmachtfreude** [comment]
I had the same issue. Documentation should be updated.
https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload
```
upload = imagekit.upload(
    file=open("image.jpg", "rb"),
    file_name="my_file_name.jpg",
    options=UploadFileRequestOptions(
        tags = ["tag1", "tag2"]
    )
)
```

**Comment by imagekitio** [comment]
Fixed in version 3.0.2

**Comment by pageinfoabout** [comment]
thanks. but i got question. how to control the size of the photo?
